ClosedXML/Excel/XLWorkbook_Save.cs

```diff
@@ -3476,7 +3477,7 @@ private void AddDifferentialFormats(WorkbookStylesPart workbookStylesPart, SaveC
             var differentialFormat = new DifferentialFormat();
 
             var diffFont = GetNewFont(new FontInfo { Font = cf.Style.Font as XLFont }, false);
-            if (diffFont?.HasChildren ?? false)
+            if (diffFont.HasChildren)
```

Comment:
Definitely keep the `?` and `??`, please. I have been working on branches that might return `null`s. Not sure if I'm going to merge them, but again, I'd rather keep the code future-proof. Please revert these changes for all the "style" variables in these methods.